documentation/src/main/asciidoc/chapter-2-set-up.asciidoc

```diff
@@ -248,13 +248,11 @@ If a policy is given for a specific mapper via `@Mapper#unmappedTargetPolicy()`,
 |`WARN`
 |===
 
-=== Using MapStruct on Java 9
+=== Using MapStruct on Java 9 and higher versions
```

Comment:
I'd suggest "Using MapStruct with the Java Module System"

Reply:
6510ece